mteb/evaluation/MTEB.py

```diff
@@ -295,6 +295,15 @@ def run(
                 logger.info(f"Loading dataset for {task.metadata_dict['name']}")
                 task.load_data(eval_splits=task_eval_splits, **kwargs)
 
+                # be sure the k_values includes the ones in the task
```

Comment:
This is where I check for other non-default k-values (like 20). It seemed like the code for this had to go in this section so it could interact with the task and the evaluator.  

I could make it specific to the abstract task though, but since we have kwargs I think it won't affect the other classes.  

Seem good or any suggestions @KennethEnevoldsen?

Reply:
Couldn't we add 20 to the default list of metrics calculated (then it would calculate it for all datasets, but I  don't think it takes too long?) - it would also avoid the added complexity here.

Reply:
I'm ambivalent.  My guess is most datasets use an existing number that is a power of 10, so I'll remove this code and we can add it back in if we see other datasets with unusual metrics (nDCG @ 30 or something).